src/modules/api.js

```diff
@@ -0,0 +1,5 @@
+const baseUrl = 'https://api.tvmaze.com/shows';
+// Generate api key => 1. beu8gYL5wu1Kf1UTc5mm 2.fZEB2tpIC45iiUMRjYli
+const involvementUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/fZEB2tpIC45iiUMRjYli/';
```

Comment:
- [Optional] It's a bad practice to store confidential information such as the API key to GitHub. Other developers may preserve your secret since Git is distributed. Let's kindly be sure to always encrypt sensitive data or ignore it using gitignore: For example, these data can be stored in an env file and ignored it.

Reply:
# Thanks for the hads up @FrederickMih .

***I understand i don't wanna make the api key visible. But in this case I am not using .env file***

🏷️ Thanks @FrederickMih 
# Happy Coding 👋 😄 
---